doc/version_history.rst

```diff
@@ -2,6 +2,15 @@
 Version History
 ===============
 
+v0.2.1
```

Comment:
Maybe make it `v0.3.0`? These seems a bit beyond a "patch" update.

Reply:
Updated in the commit: [b244b0a](https://github.com/lsst-ts/ts_m2com/pull/3/commits/b244b0aad8f4c2a0d09a08aeac6524d5ec86b72e). Thanks!